### ISSUE TITLE
Just download Krita app. Don't see plugin folder.

### ISSUE THREAD

**Issue by tomyHHC** [Acly/krita-ai-tools#13]
Is the instruction here outdated? 

May someone help me know how to install this awesome plugin tool? 

Thanks!

**Comment by gohan2091** [comment]
The instructions to install this is not very clear. Installation folder? what is that? Afrer 5 minutes or playing around I figured out its (for me at least): C:\Program Files\Krita (x64) and not C:\Program Files\Krita (x64)\bin or the plugins folder.

**Comment by tommywright** [comment]
I agree.. that "Krita installation folder" was a head scratcher. So you just place this folder "krita-ai-tools-main" into the folder with Krita(x64)\ I guess? Also, how do you know that the plugin is installed correctly?
